source/documentation/get-message.html.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: Get all messages for an excise movement
+weight: 60
+---
+
+# Get a message belonging to an excise movement
+<%= partial 'documentation/partials/google_tag_manager_js' %>
+Use the GET endpoint [Get a messages for an excise movement using the movement ID and messageId](/api-documentation/docs/api/service/excise-movement-control-system-api/1.0/oas/page#tag/Movements/operation/getMessage) to get a message belonging to a specific excise movement.
```

Comment:
Should this be "Get a message for an excise movement using the movement ID and message ID (movement ID and messageId are inconsistent way of saying the ID. can we pick one or the other please?